doc/Tribler REST API.md

```diff
@@ -79,6 +80,26 @@ Returns all discovered channels in Tribler.
 }
 ```
 
+## `GET /channels/discovered/{channelcid}/torrents`
+
+Returns all discovered torrents in a specific channel.
+
+### Example response
+
+```json
+{
+    "torrents": [{
+        "id": 4,
+        "infohash": "97d2d8f5d37e56cfaeaae151d55f05b077074779",
+        "name": "Ubuntu-16.04-desktop-amd64",
+        "length": 8592385,
```

Comment:
True, I've added a description about the unit of the torrent size. I also renamed the `length` key to `size` which I think is more clear.